charts/dvpe-deployment-gloo/README.md

```diff
@@ -61,7 +61,7 @@ This secret needs to contain the client_secret information as base64 encoded str
 Gloo expects a trailing newline at the secret string.
 You can encode your secret with the following command:
     ```bash
-    echo 'clientSecret: <clientSecretFromIDP>' | base64
+    echo -n 'clientSecret: <clientSecretFromIDP>' | base64
```

Comment:
good point!

Reply:
Gloo expectes a secret with a new line. `echo -n ` will not work